### PR TITLE
Pin direct dependencies on master

### PR DESCRIPTION
**Pull request by arnaudsjs**
# Description

Pin direct dependencies in the `requirements.txt` file

Part of inmanta/irt#406

# Self Check:

- [x] Attached issue to pull request
- [ ] ~~Changelog entry~~

### CODE REVIEW

**Comment by arnaudsjs** [comment]
@wouterdb The settings of this repo require an approval before it can be merged.

**Review by wouterdb — APPROVED** [review]

**Pull request merged**
